base_layer/wallet/src/transaction_service/protocols/transaction_broadcast_protocol.rs

```diff
@@ -205,31 +215,36 @@ where TBackend: TransactionBackend + Clone + 'static
                         self.timeout = to;
                         info!(
                             target: LOG_TARGET,
-                            "Broadcast monitoring protocol (Id: {}) timeout updated to {:?}", self.id ,self.timeout
+                            "Transaction monitoring event 'updated_timeout' triggered (Id: {}), timeout updated to {:?}", self.id ,self.timeout
+                        );
+                    } else {
+                        trace!(
+                            target: LOG_TARGET,
+                            "Transaction monitoring event 'updated_timeout' triggered (Id: {}) ({:?})",
+                            self.id,
+                            updated_timeout,
                         );
                     }
                 },
                 () = delay => {
-                },
-            }
-
-            info!(
```

Comment:
This also feels like it's an important event. Perhaps even should be a `warn!`

Reply:
This was just moved to only log if it was a timeout where as before anything that exited the select! would fire this log. Don't think this is a warn because it is not a show stopper and base nodes can be busy so we try again?

Reply:
> This also feels like it's an important event. Perhaps even should be a `warn!`

It is only informative for trace level debugging, as the broadcast-to-mempool-and-wait loop will be repeated.